YaqeenDAL/Model/Model.cs

```diff
@@ -196,6 +197,27 @@ public class Interest : Entity
         public virtual ICollection<Content> Contents { get; set; }
     }
 
+    public class Medication : Entity 
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int MedicationId { get; set; }
+        public string PatientId { get; set; }
+        public string MedicationType { get; set; }
+        public string MedicationName { get; set; }
+        public string? Photo { get; set; }
+        public int Dosage { get; set; }
```

Comment:
You probably want to add unit as well (as enum).
Dosage might need to be a float (I hate floats though).. if we want to say 1.5 Tbsp for example